stable/jenkins/values.yaml

```diff
@@ -235,6 +235,10 @@ master:
   # List of plugins to install in addition to those listed in master.installPlugins
   additionalPlugins: []
 
+  # Enable to initialize the Jenkins master only once on initial installation.
+  # Without this, whenever the master gets restarted (Evicted, etc.) it will fetch plugin updates which has the potential to cause breakage.
+  initializeOnce: false
```

Comment:
From Slack chat: please add a note that the `master.initializeOnce` option it will only work if `master.persistence` is set to `true` (otherwise when a pod is rescheduled, the plugins won't be present)

Reply:
This is now done, kindly see comment above:
```
# Note that for this to work, `persistence.enabled` needs to be set to `true`
```